alws/crud/test.py

```diff
@@ -3,18 +3,34 @@
 import re
 import urllib
 from io import BytesIO
+
+from sqlalchemy import and_
 from sqlalchemy.future import select
 from sqlalchemy.orm import Session, selectinload
 from sqlalchemy.sql.expression import func
 
 from alws import models
 from alws.config import settings
-from alws.constants import TestTaskStatus
+from alws.constants import BuildTaskStatus, TestTaskStatus
 from alws.schemas import test_schema
 from alws.utils.pulp_client import PulpClient
 from alws.utils.file_utils import download_file
 from alws.utils.parsing import parse_tap_output, tap_set_status
 
+async def create_test_tasks_for_build_id(db: Session, build_id: int):
+    async with db.begin():
+        ## We get all build_tasks with the same build_id
+        ## and whose status is COMPLETED
+        build_task_ids = (await db.execute(
+            select(models.BuildTask.id).where(and_(
```

Comment:
`and` is default grouping of conditions in SQLAlchemy, so you can skip it in simple requests